packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -136,11 +136,21 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
   @Prop() shortHeading: string;
 
   /** The component's text. */
+  @Prop({ reflect: true }) heading: string;
```

Comment:
Let's not reflect `heading`, as it is content. I don't recall why we were originally reflecting `textLabel`. 🤔